shared/src/main/scala/Parser.scala

```diff
@@ -18,7 +18,10 @@ object Parser:
   type ParserRet[T] = Either[VyxalCompilationError, T]
 
   private def toValidName(name: String): String =
-    name.filter(_.isLetterOrDigit).dropWhile(!_.isLetter)
+    name
+      .filter(c => c.isLetterOrDigit || c.toString == "ı")
```

Comment:
```suggestion
      .filter(c => c.isLetterOrDigit || c == 'ı')
```

Reply:
The joys of still getting used to `char` and `String` not being interchangeable :p